utils/service/cron_schedule_message.ts

```diff
@@ -1,16 +1,13 @@
 import { CronJob } from "cron";
 import {
   CRON_EVERY_MONDAY_AT_10,
-  CRON_START_OF_QUARTER,
   CRON__TWOWEEKS_AFTERSTART_OF_QUARTER,
   SHUFFLE_SETTINGS_ID,
   TORONTO_TIME_ZONE_IDENTIFIER,
 } from "../constants/consts.ts";
 import { find_shuffle_setting } from "../controllers/shuffle-bot-groups.ts";
 
-import { post_networking_calendar } from "../../src/Events/post_networking_calendar.ts";
```

Comment:
missed removing this import from this [PR](https://github.com/tech-tankorg/tech-tank-slackbot/pull/157) so removing it here too